docs/sphinx/ome-tiff/specification.rst

```diff
@@ -434,3 +438,96 @@ The master file containing the full OME-XML metadata should be either an
 OME-XML companion file with the extension :file:`.companion.ome` or a master
 OME-TIFF file containing the full metadata (see :ref:`multifile_samples` for
 representative samples).
+
+Pyramidal OME-TIFF
+------------------
+
+.. versionadded:: 6.0.0
+
+The OME-TIFF specification supports multi-resolution images or pyramidal
+images where individual planes are stored at different levels of resolution.
+The intermediate downsampled image planes are called pyramidal levels,
+sub-resolution image planes or sub-resolutions.
+
+Supported resolutions
+^^^^^^^^^^^^^^^^^^^^^
+
+OME-TIFF planes can be reduced alongside the X and Y dimensions. Each pyramidal
+level must be a downsampling of the full-resolution plane in the X and Y
+dimensions and the resolution should stay unchanged in the other dimensions.
+The downsampling factor should be an integer value, identical alongside the X
```

Comment:
These three constraints might be better placed in a bullet list.  While the *should* qualifier is appropriate, they are otherwise unrelated requirements.